src/test/java/com/epam/tc/hw3/pages/components/ColorsDropDownComponent.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.tc.hw3.pages.components;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ColorsDropDownComponent {
+
+    @FindBy(xpath = ".//*[@class='colors']//*[@class='uui-form-element']")
+    private WebElement colorsDropDown;
+
+    @FindBy(xpath = ".//*[@class='colors']//*[@class='uui-form-element']//option[text() = 'Yellow']")
+    private WebElement yellowColor;
```

Comment:
There are more suitable solutions and locators

Reply:
done